app/src/main/java/org/ometa/lovemonster/ui/activities/UserLoveActivity.java

```diff
@@ -75,7 +75,12 @@ private void setupMakeLoveButton() {
         makeLoveButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                final MakeLoveDialogFragment makeLoveDialogFragment = MakeLoveDialogFragment.newInstance(user);
+                final MakeLoveDialogFragment makeLoveDialogFragment;
+                if (user.equals(currentUser)) {
+                    makeLoveDialogFragment = MakeLoveDialogFragment.newInstance(new User("foo@example.com", ""));
```

Comment:
That constraint is probably overly restrictive, considering we only ever really need a username for stuff to work.

Probably is safe to remove the requirement on email address being there, and only require username.